index.html

```diff
@@ -688,11 +688,18 @@ <h4>
         The [=remote end steps=] are:
       </p>
       <ol class="algorithm" data-cite="WEBDRIVER2">
+        <li>Let |posture| be the result of invoking get a property "posture"
```

Comment:
Furthermore, "get a property" will give you some JavaScript value that can technically be of any type (including undefined if it was not passed in the HTTP request). You also need to ensure it's a string. See e.g. step 2 of the remote end steps of https://w3c.github.io/webdriver/#dfn-dispatch-actions-for-a-string or step 2 of https://w3c.github.io/webdriver/#send-alert-text.

Reply:
Would you know how to make the link? It should point to this https://w3c.github.io/webdriver/#dfn-getting-properties

In case of https://www.w3.org/TR/webdriver2/#dfn-extension-commands, `[=extension commands=]` works.

But in this case `[=getting properties=]` doesn't work. It is defined in here https://github.com/w3c/webdriver/blob/master/index.html#L362C18-L363C36
```
<dfn data-lt="getting properties|getting the
 property">getting a property</dfn>

```
I have tried also 
[=getting the property=]
[=getting a property=]

Reply:
The problem's that the `<dfn>` is not exported by the WebDriver spec. This has also [affected the Generic Sensor spec](https://github.com/w3c/sensors/blob/d8b0f67c3acc27330f97554185a32577a7d9d063/index.bs#L55).

In ReSpec, the easiest workaround is to use the [`data-cite` attribute](https://respec.org/docs/#data-cite).

Reply:
Fixed in https://github.com/JuhaVainio/W3C-device-posture/pull/1/commits/6bca850082316fb3e17953e1a95888300cef6569

Reply:
https://github.com/JuhaVainio/W3C-device-posture/pull/1/commits/0561e510aeaa4711062f53c7c45ddaa5511f6e7a added to fulfill the "need to ensure it's a string" requirement